Src/Support/GoogleApis.Tests/Apis/Download/MediaDownloaderTest.cs

```diff
@@ -314,11 +314,12 @@ public void Download_Error_JsonResponse()
         }
 
         [Test]
-        public void Download_Error_PlaintextResponse()
+        [TestCase("Not Found")]
+        [TestCase("")]
+        public void Download_Error_PlaintextResponse(string responseText)
         {
             var downloadUri = "http://www.sample.com";
             var chunkSize = 100;
-            var responseText = "Not Found";
 
             var handler = new MultipleChunksMessageHandler { ErrorResponse = responseText };

```

Comment:
Just want to verify that it will result in parsedResponse = null in the code below(line 49 in MediaApiErrorHandling).
I can't see it immediately (since empty string is different from null).
Am I missing something?